code.gs

```diff
@@ -1937,6 +1937,37 @@ var i18n = {
     'Other email': 'Jiné e-mailové adresy',
     'It looks like you are using an outdated version of this script': 'Vypadatá to, že používáte zastaralouv verzi skriptu',
     'You can find the latest one here': 'Poslední verzi najdete zde',
+  },
+    'fa': {
```

Comment:
Here four spaces should be two to fit the [semistandard linting rules](https://github.com/GioBonvi/GoogleContactsEventsNotifier/blob/master/.github/CONTRIBUTING.md#coding-guidelines).
```suggestion
  'fa': {
```